webb-tools/relayer
# Fees for substrate

## Summary of changes
- Add fee and refund logic for substrate
- Update test instructions to include tangle setup
- Update fee endpoints
- Update test case `should withdraw using private transaction` to use fee and refund


## Note
We have updated the fee endpoint to support requests for  substrate and evm based chains, following are the new points
- EVM:
```
  /api/v1/fee_info/evm/:chain_id/:vanchor/:gas_amount
```
-  Substrate
```
 /api/v1/fee_info/substrate/:chain_id/:partialFee
```

### Reference issue to close (if applicable)
- Closes #364 
- Closes #503 
-----
### Code Checklist
